sdk/go.mod

```diff
@@ -1,9 +1,58 @@
 module github.com/pulumi/pulumi-rancher2/sdk/v3
 
-go 1.16
+go 1.17
```

Comment:
Should this be go 1.18?

Reply:
No, the SDK is an external thing and we should specify the minimum go version we support (go 1.17). The provider is internal code and is part of a binary so that can be any version of go that we want